src/components/Confirm/index.js

```diff
@@ -60,7 +60,7 @@ const Previews = localised(({capture, retakeAction, confirmAction, error, method
     <div className={classNames(style.previewsContainer, theme.fullHeightContainer, {
       [style.previewsContainerIsFullScreen]: isFullScreen,
     })}>
-      { error.type ? <Error {...{error, withArrow: true}} /> :
+      { error.type ? <Error {...{error, withArrow: true, role: "alert", focusOnRender: false}} /> :
```

Comment:
@stefaniacardenas I know I have added this prop on the component earlier, however now I think the naming is a bit misleading - it should be `focusOnMount` rather than `focusOnRender`, as it has a bit of a different meaning 🙃

Reply:
Cool, I've renamed it 👍